addons/openebs/2.6.0/operator.yaml

```diff
@@ -0,0 +1,791 @@
+# adapted from https://github.com/openebs/charts/blob/gh-pages/2.6.0/openebs-operator.yaml
+# 1. rbac.authorization.k8s.io/v1beta1 -> rbac.authorization.k8s.io/v1
+# 2. remove namespace
+# 3. set OPENEBS_IO_JIVA_REPLICA_COUNT to 0
+# 4. set OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG to "false"
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
+# NOTE: On GKE, deploy the openebs-operator.yaml in admin context
+
+# Create Maya Service Account
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openebs-maya-operator
+  namespace: openebs
```

Comment:
Namespace -> I would expect all these values to be the template value from the kurl spec. If I compare to 1.12, all the ns fields were removed in 1.12.